src/app/workflow/view/view.component.css

```diff
@@ -13,3 +13,11 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+
+/* Drop-down menu restyling to match new design: bolded text & smaller height */
```

Comment:
Oddly enough, this does appear to be a unique dropdown (different than navbar top right, different than search)